.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
What is the reason for adding this file? It looks like a personal environment configuration file. If you don't need it, please delete it.

Reply:
fixed by https://github.com/Yamato-Security/hayabusa/pull/876/commits/b82087ba6bedfddc656f1faeb130583090bdbed7